test/resources/demo-users/user/ns2/ec-integration-test.yaml

```diff
@@ -17,7 +17,7 @@ spec:
   resolverRef:
     params:
       - name: url
-        value: 'https://github.com/gbenhaim/build-definitions'
+        value: 'https://github.com/konflux-ci/build-definitions'
       - name: revision
         value: ec-pub-key
```

Comment:
The revision looks strange. This should be a branch on the upstream repo, but currently  it looks to me as the name given to a pr branch on contributor's fork. Please mqnually test locally per my comment in Slack to verify.